inc/config.class.php

```diff
@@ -2904,88 +2904,34 @@ static function getConfigurationValues($context, array $names = []) {
    /**
     * Load legacy configuration into $CFG_GLPI global variable.
     *
-    * @param boolean $older_to_latest Search on old configuration objects first
-    *
     * @return boolean True for success, false if an error occured
+    *
+    * @since 10.0.0 Parameter $older_to_latest is not longer used.
     */
-   public static function loadLegacyConfiguration($older_to_latest = true) {
+   public static function loadLegacyConfiguration() {
 
       global $CFG_GLPI, $DB;
 
-      $config_tables_iterator = $DB->listTables('glpi_config%');
-      $config_tables = [];
-      foreach ($config_tables_iterator as $config_table) {
-         $config_tables[] = $config_table['TABLE_NAME'];
-      }
-
-      $get_prior_to_078_config  = function() use ($DB, $config_tables) {
-         if (!in_array('glpi_config', $config_tables)) {
-            return false;
-         }
+      Config::forceTable('glpi_configs');
```

Comment:
This one seems useless now.
```suggestion
      Config::forceTable('glpi_configs');
```